core/src/main/scala/com.snowplowanalytics.stream/loader/clients/BulkSender.scala

```diff
@@ -74,13 +75,15 @@ trait BulkSender[A] {
 }
 
 object BulkSender {
+  implicit val contextShift: ContextShift[IO] = IO.contextShift(ExecutionContext.global)
```

Comment:
Why do we need this now, but didn't need before?

Reply:
Good question. Could it be that snowplow-badrows 2.1.0 comes with latest Iglu client that brings cats-effect > 2.0.0 ? I'm removing cats from the dependencies as this is brought as a transitive dependency.

Reply:
Yeah, you're right `IO.fromFuture` requires `CS` since 2.0.0: https://github.com/typelevel/cats-effect/commit/ebab1db7b9a116c939c60213339743d5b9907ac9#diff-c01f002766a28f49dfe20d6b1f0afd50R1203-R1204.

And good call on removing transitive dependencies.